### PR TITLE
likes, retweets total changed to average

### PR DESCRIPTION
**Pull request by E-STAT**

### CODE REVIEW

File: api/query.py [hunk 1]
```diff
@@ -118,8 +118,8 @@ async def twitterOverview(self, asaID: str) -> TwitterOverview:
             await Twitter.filter(asa_id=asaID)
             .annotate(
                 sentimentTotal=Avg("sentiment_score"),
-                retweetTotal=Sum("retweets"),
-                likeTotal=Sum("likes"),
+                retweetTotal=Sum("retweets")/Count("retweets"),
```
Comment (Iambusayor) [inline api/query.py:121]: This computation is done on the server. Use Avg to perform it on the backend, as I did for the sentiment score.

File: api/query.py [hunk 2]
```diff
@@ -118,8 +118,8 @@ async def twitterOverview(self, asaID: str) -> TwitterOverview:
...
+                likeTotal=Sum("likes")/Count("likes"),
```
Comment (Iambusayor) [inline api/query.py:122]: same as above

**Review by OladetounJed — DISMISSED** [review]
lgtm

**Review by Iambusayor — CHANGES_REQUESTED** [review]

File: api/query.py (hunk 1 above)
Comment (E-STAT) [reply api/query.py:121]: Oh yeah, got that!

**Comment by Iambusayor** [comment]
> lgtm

😒

**Review by Iambusayor — APPROVED** [review]
lgtm

**Pull request merged**